### PR TITLE
solaris based systems priv api flags updates

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/unix/solarish/mod.rs [hunk 1]
```diff
@@ -2233,6 +2233,14 @@ pub const __PROC_PROTECT: ::c_uint = 0x0008;
 pub const NET_MAC_AWARE: ::c_uint = 0x0010;
 pub const NET_MAC_AWARE_INHERIT: ::c_uint = 0x0020;
 pub const PRIV_AWARE_RESET: ::c_uint = 0x0040;
+pub const PRIV_XPOLICY: ::c_uint = 0x0080;
+pub const PRIV_PFEXEC: ::c_uint = 0x0080;
```
Comment (pfmooney) [inline src/unix/solarish/mod.rs:2237]: Shouldn't this be `0x0100`?

**Review by pfmooney — CHANGES_REQUESTED** [review]

File: src/unix/solarish/mod.rs (hunk 1 above)
Comment (devnexen) [reply src/unix/solarish/mod.rs:2237]: good catch bad c/p :)

Comment (jclulow) [reply src/unix/solarish/mod.rs]: Did you run the tests?  They generally catch transcription errors like this.

Comment (devnexen) [reply src/unix/solarish/mod.rs]: it does I do in other platforms but in my illumos VM tests get stuck and mostly I have to reboot the VM.

**Pull request merged**